sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Azure.Monitor.OpenTelemetry
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Monitor OpenTelemetry in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class AzureMonitorOpenTelemetryExtensions
+    {
+        /// <summary>
+        /// Adds Azure Monitor OpenTelemetry into service collection.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="options">The <see cref="AzureMonitorOpenTelemetryOptions" /> instance for configuration.</param>
+        /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+        public static IServiceCollection AddAzureMonitorOpenTelemetry(this IServiceCollection services, AzureMonitorOpenTelemetryOptions? options = null)
```

Comment:
Has there been any discussion about the naming? For example if we expect all the current users of ApplicationInsights SDK to eventually move to this SDK, do we expect them to learn or care what 'OpenTelemetry' is?

I want to ensure whatever name we are using aligns the overall plan for how the Azure Monitor product team wants to refer to it in Azure Portal pages, marketing material, documentation, blogs, conferences, etc.

Reply:
Product team worked with Azure SDK and confirmed this name. The documentation for documents, blogs are all different at this point.